### PR TITLE
New package: bitwarden-1.29.1

### PR DESCRIPTION
**Pull request by AI-nsley69**
#### General
- [x] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/bitwarden/template [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+pkgname=bitwarden
+version=1.29.1
+revision=1
+archs="x86_64"
+depends="nss libsecret libnotify libXtst libXScrnSaver"
+short_desc="Desktop app for Bitwarden password manager"
+maintainer="AI-nsley69 <trainsley69@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://bitwarden.com/"
+distfiles="https://github.com/bitwarden/desktop/releases/download/v${version}/Bitwarden-${version}-amd64.deb"
```
Comment (classabbyamp) [inline srcpkgs/bitwarden/template:11]: why use the x86_64-only deb? Is it not possible to build from source?

Comment (AI-nsley69) [reply srcpkgs/bitwarden/template:11]: I couldn't figure out how to build it from source, the instructions seems like it'll only compile typescript to javascript, then run it through `npm run electron`.

Comment (classabbyamp) [reply srcpkgs/bitwarden/template:11]: looks like they have a [build workflow](https://github.com/bitwarden/desktop/blob/master/.github/workflows/build.yml) for github ci. maybe that'll shed some light on it

Comment (AI-nsley69) [reply srcpkgs/bitwarden/template:11]: I'll give it a shot!

**Pull request closed without merging**